common4j/src/main/com/microsoft/identity/common/java/util/StringUtil.java

```diff
@@ -342,6 +342,10 @@ public static String encodeUrlSafeString(@NonNull final String stringToEncode) {
         return Base64.encodeToString(toByteArray(stringToEncode), Base64.NO_WRAP | Base64.NO_PADDING | Base64.URL_SAFE);
     }
 
+    public static String decodeUrlSafeString(@NonNull final String stringToDecode) {
```

Comment:
probably add some tests for this method

Reply:
Not needed at the moment, so I have removed it.